tests/rbd/rbd_utils.py

```diff
@@ -75,6 +76,18 @@ def random_string(self):
         temp_str = "".join([random.choice(string.ascii_letters) for _ in range(10)])
         return temp_str
 
+    def initial_rbd_config(self, rbd, pool, image, **kw):
+        size = kw.get("size")
+        if not rbd.create_pool(poolname=pool):
+            # create pool does not catch exceptions, it returns true/false.
+            # so we are returning 1 instead of raising exception
+            return 1
+
+        rbd.create_image(pool_name=pool, image_name=image, size=size)
+        if kw.get("client"):
+            client = kw.get("client")
+            run_fio(image_name=image, pool_name=pool, client_node=client)
+
```

Comment:
1. fix the comments.
2. clients are optional here, if test case says to execute i/o, then only we can pass the clients and run i/o.
3. Size is kept flexible and expected from individual suite, so that if ant other test case want to create image with different size, it will helpful.
4. Added the method doc

Reply:
2. Removed the fio from method to align with the purpose of initial rbd configuration.
3. added the default value for size.